NuGet.Config

```diff
@@ -3,8 +3,12 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <add key="local" value="../../Dolittle/Packages" />
```

Comment:
@einari Shouldn't we remove this?

Reply:
:) Yup. I accidently checked the file in.  I use this locally to deploy faster when working.
It will give warnings during build which is annoying to look at..